Yoast/wordpress-seo
# CS: various doc fixes

## Context

* Code documentation consistency 

## Summary

This PR can be summarized in the following changelog entry:

* Code documentation consistency 

## Relevant technical choices:

* Docs: fix various minor formatting and punctuation issues
* Docs: remove redundant var name in `@var` tags in property docblocks.
* Docs: remove redundant var name in `@return` tags
* Docs: fix tag ordering
* Docs: fix incorrect/incomplete type information


## Test instructions

This PR can be tested by following these steps:
* _N/A_
    This is a documentation-only change and should have no effect on the functionality. If the build passes (linting, test runs), we're good.
